### DISCUSSION TITLE
Add an option to Get-ChildItems to ingore symlinks when using -Recurse

### DISCUSSION THREAD

**Discussion by tuscen** [PowerShell/PowerShell#16855] [Enhancements]
Sometimes I need to clear `obj` and `bin` folders in my C# projects. To do this I usually traverse my repos recursively to find those folders to delete them, but apparently symlinks really mess with this process. 

I use pnpm as package manager for frontend and it uses symlinks for package deduplication. So when I try to recursively delete those folders, `Get-ChildItems` follows symlinks and tries to find them in the pnpm package directory which sometimes is quite big and deep. As a result it ends with a very long recursive search. I work around this by limiting depth to at most a few directories where I know `obj` and `bin` folders must be.

It would be very handy for `Get-ChildItems` to have an option to not follow symlinks

**Comment by iSazonov** [comment]
Get-ChildItem doesn't recurse into symlinks without FollowSymlink switch but there is a bug #9126.
